docs/mobile/game-session/game_session_db_design.md

```diff
@@ -0,0 +1,53 @@
+# DynamoDB Design
+
+## Entities
+
+### Config
+
+This entity contains the global settings and configurable parts of the application. The list of items included here is as follow
+
+- Points granted to each team estimated the right popular trick answer
+- Points granted per to teams guessed the right answer
+
+### GameSession
+
+The higher level entity contains all information related to a game from start to finish.
+
+- Game: A game object that comes from Aurora. It's a snapshot of the data when the game starts
+- Teams: An array of team entities that involved in the game
+- StartedDateTimeUTC: Date time of when the game started in UTC
+- MultiDeviceMode: Boolean indicating if it's a multi-device with facilitator or not
```

Comment:
Maybe: "Boolean indicating multi-device scenario" (if True, then one device among a team's multiple devices will be randomly assigned the Facilitator; if False, then there is only one device per team and that device will also have Facilitator privileges)

Reply:
Variables configured on the teacher side that affect student game sessions include (variable names are just draft suggestions):
- VotingTime: Amount of time that voting teams have to vote (this equals the amount of time that the team coming up with the trick answers (aka QuestionTeam?) has to come up with and select their trick answers.
- HintsRevealTime: Amount of time that passes before hints are revealed. This must be less than VotingTime.
- We should set default values for both these variables and allow teachers to adjust them before a game starts. VotingTime should be something that a teacher can override in the middle of game (e.g., if a teacher sees that all teams are ready to proceed, the teacher should be able to move on to the results screens without having to wait for the entire VotingTime to have passed).

Reply:
Good point, per our discussion, I added `VotingTime` and `HintsRevealTime` to `Config` also and also here. Regarding `VotingTime` modification at runtime, we should probably just do this through UI so teachers can end it by just tapping a button.